Invoices.elm

```diff
@@ -24,45 +25,50 @@ type alias Product =
     , unit : Int
     , tax : Float
     , price : Float
-    , total : Float
```

Comment:
Total attr was redundant as it's really not part of a day but rather a function of data `Product -> Float`